src/server/scripts/Northrend/AzjolNerub/ahnkahet/boss_herald_volazj.cpp

```diff
@@ -382,6 +438,74 @@ class spell_herald_volzaj_insanity : public SpellScriptLoader
     }
 };
 
+// 60291 Volazj Whisper: Aggro
+// 60292 Volazj Whisper: Insanity
+// 60293 Volazj Whisper: Slay 01
+// 60294 Volazj Whisper: Slay 02
+// 60295 Volazj Whisper: Slay 03
+// 60296 Volazj Whisper: Death 01
+// 60297 Volazj Whisper: Death 02
+class spell_volazj_whisper : public SpellScriptLoader
+{
+public:
+    spell_volazj_whisper() : SpellScriptLoader("spell_volazj_whisper") { }
+
+    class spell_volazj_whisper_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_volazj_whisper_SpellScript);
+
+        bool Validate(SpellInfo const* /*spell*/) override
+        {
+            return ValidateSpellInfo(
+            {
+                SPELL_WHISPER_AGGRO,
+                SPELL_WHISPER_INSANITY,
+                SPELL_WHISPER_SLAY_1,
+                SPELL_WHISPER_SLAY_2,
+                SPELL_WHISPER_SLAY_3,
+                SPELL_WHISPER_DEATH_1,
+                SPELL_WHISPER_DEATH_2
+            });
+        }
+
+        bool Load() override { return GetCaster()->GetTypeId() == TYPEID_UNIT; }
+
+        void HandleScriptEffect(SpellEffIndex /* effIndex */)
+        {
+            Unit* target = GetHitPlayer();
+            Creature* caster = GetCaster()->ToCreature();
+            if (!target || !caster)
+            {
+                return;
+            }
+
+            uint32 text = 0;
+            switch (GetSpellInfo()->Id)
+            {
+                case SPELL_WHISPER_AGGRO:    text = WHISPER_AGGRO;    break;
+                case SPELL_WHISPER_INSANITY: text = WHISPER_INSANITY; break;
+                case SPELL_WHISPER_SLAY_1:   text = WHISPER_SLAY_1;   break;
+                case SPELL_WHISPER_SLAY_2:   text = WHISPER_SLAY_2;   break;
+                case SPELL_WHISPER_SLAY_3:   text = WHISPER_SLAY_3;   break;
+                case SPELL_WHISPER_DEATH_1:  text = WHISPER_DEATH_1;  break;
+                case SPELL_WHISPER_DEATH_2:  text = WHISPER_DEATH_2;  break;
+                default: return;
+            }
+            caster->AI()->Talk(text, target);
+        }
+
+        void Register() override
+        {
+            OnEffectHitTarget += SpellEffectFn(spell_volazj_whisper_SpellScript::HandleScriptEffect, EFFECT_0, SPELL_EFFECT_SCRIPT_EFFECT);
+        }
+    };
+
+    SpellScript* GetSpellScript() const override
+    {
+        return new spell_volazj_whisper_SpellScript();
+    }
+};
+
```

Comment:
I'm sorry about this, i forgot to add its original author too -> https://github.com/offl